src/Symfony/Component/Console/Output/StreamOutput.php

```diff
@@ -91,9 +91,15 @@ protected function doWrite($message, $newline)
      */
     protected function hasColorSupport()
     {
+        $colorEnv = getenv('ANSI');
+        if ('' !== $colorEnv) {
```

Comment:
When the env var is not set, `getenv` returns `false`, not an empty string. I guess we want to force ANSI/no-ANSI as soon as the env var is set, right?